omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx.conf.erb

```diff
@@ -12,10 +12,17 @@ events {
 }
 
 http {
+<%
+    time_format = node['private_chef']['nginx']['time_format']
+    if time_format != 'time_iso8601' and time_format != 'time_local'
```

Comment:
If I am understanding your logic correctly, consider using `&&` instead of `and`.  Examples:
```# Use &&/|| for boolean expressions, and/or for control flow. (Rule of thumb: If you have to use outer parentheses, you are using the wrong operators.)

# boolean expression
if some_condition && some_other_condition
  do_something
end

# control flow
document.saved? or document.save!
```

Reply:
Also, you might consider something like this instead of your if statement (WARNING: untested):
```
# assumes time_format is already defined, and should only be 'time_local' or 'time_iso8601'
time_format == 'time_local' or time_format = 'time_iso'
```

Reply:
Thanks for the suggestions! I will make the changes.